recipe/meta.yaml

```
@@ -13,15 +13,17 @@ source:
   sha256: {{ tar_gz_hash }}
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
     - cmake
     - {{ compiler("fortran") }}        # [linux or osx]
     - make                             # [linux or osx]
+    - m2w64-make                       # [win]
```

```suggestion
    - make 
```
The `make` feedstock now supports Windows so that should be all that is needed here.